### ISSUE TITLE
hydro-link issues in .cshtml partials

### ISSUE THREAD

**Issue by westonwalker** [hydrostack/hydro#7]
Sometimes when adding a hydro-link attribute, the link becomes broken with a "/null" path. (ex https://localhost:7231/null when it should be https://localhost:7231/jobs).

The 2 scenarios I've had this happen in are:

1. An anchor tag in a normal .cshtml partial (not extending HydroComponent). For example, I have a _NavBar.cshtml partial for my Navbar. It's very dumb, just abstracts some markup. The link in this will return https://localhost:7231/null.
```
<nav class="w-full bg-base-200">
	<div class="py-6 px-8 max-w-7xl mx-auto">
		<div class="flex justify-center items-center">
			<div>
				<a href="/" hydro-link>
					Home
				</a>
			</div>
		</div>
	</div>
</nav>
```
2. A for loop in a .cshtml page. I have a page that loops through job openings. Each loop builds an anchor tag for that jobs page. Adding hydro-link will result in https://localhost:7231/null
```
@foreach (var job in Model.Jobs)
{
  <a href="/jobs/@job.Slug">Link to job</a>
}
```

**Comment by kjeske** [comment]
Are threse urls plain strings passed to the `href` attribute (`/` and `/jobs/@job.Slug`) or you're using `Url.Page` to generate them?

**Comment by westonwalker** [comment]
plain url strings

**Comment by westonwalker** [comment]
Plain url strings

**Comment by westonwalker** [comment]
Adding some more findings:

The markup in the browser has the correct url in the href attribute. However, when clicking the link, Hydro is sending me to the "/null" page.

**Comment by kjeske** [comment]
Possible fix is in version 0.8.4

**Comment by westonwalker** [comment]
Both scenarios I listed above are fixed after upgrading to 0.8.4.
